src/utils/seo.cr

```diff
@@ -0,0 +1,3 @@
+record SEO,
+  page_title : String = "Lucky - Crystal Web Framework",
```

Comment:
I don't think there should be a default. That way we will know at compile time if something is missing a page title